### DISCUSSION TITLE
No tests found

### DISCUSSION THREAD

**Discussion by nick-nds** [nvim-neotest/neotest#246] [Q&A]
I am using this plugin in the Laravel application. I have followed the installation process very closely. Yet when I run `lua require('neotest').run.run()` , I get No test found. Any idea how to debug this?

**Comment by rcarriga** [comment]
Please open a bug report with the report information filled out

**Comment by oyediyildiz** [comment]
> I am using this plugin in the Laravel application. I have followed the installation process very closely. Yet when I run `lua require('neotest').run.run()` , I get No test found. Any idea how to debug this?

I got the same thing when testing go code. In my case, the problem was that I didn't have the treesitter parser for go.

This solved it for me:
```
{
    "nvim-treesitter/nvim-treesitter", 
    build = ":TSUpdate",
    config = function()
        require("nvim-treesitter.configs").setup({
            ensure_installed = { "go" }
        })
    end
},
```
